lighthouse-core/config/constants.js

```diff
@@ -42,6 +42,7 @@ const throttling = {
 /** @type {LH.Config.Settings} */
 const defaultSettings = {
   output: 'json',
+  maxWaitForFcp: 30 * 1000,
```

Comment:
open to other defaults, 20s maybe? not sure what's reasonable

Reply:
10s is fine IMO.

Reply:
That'll probably be too low when using real throttling though, no? :/

how about 15?